.eslintrc.js

```diff
@@ -35,7 +35,15 @@ module.exports = {
     "func-names" : ["warn", "as-needed"],
     "no-underscore-dangle": ["error", {
       "allowAfterThis": true
-    }]
+    }],
+    "quotes" : ["error", "single", { "allowTemplateLiterals": true }],
+    "arrow-parens" : "off",
+    "arrow-body-style" : "off",
+    "func-names" : "off",
+    "prefer-arrow-callback" : "off",
+    "no-underscore-dangle" : "off",
+    "no-unused-vars" : "off",
```

Comment:
In my experience, this rule is very helpful in cleaning up my code.  The only example I can think of that makes it annoying is ExpressJS error handlers, where functions must have arity 4 (and so you might not use `next`, but are required to pass it in).

What cases are you finding that it would be useful to turn it off?